source/app/store/entities/documents/actions.js

```diff
@@ -258,6 +259,21 @@ export const addRedactions = createAction(
   }
 );
 
+export const cancelRedaction = createAction(
+  CANCEL_REDACTION,
```

Comment:
Just curious on the naming. Is there a specific reason to choose CANCEL over CLEAR?

Reply:
I'm good with "clear" as well. Will change

Reply:
Done